.github/workflows/add_version.yml

```diff
@@ -0,0 +1,198 @@
+name: version add
+on: 
+  workflow_dispatch:
+    inputs:
+      chain:
+        description: "Chain to update"
+        required: true
+        default: ""
+        type: string
+
+      tag-height-proposal:
+        description: "Tag, Height and Proposal. If the upgrade is not consensus break, leave height and proposal to -1."
```

Comment:
..is not *consensus-breaking,* leave...
or
...does not break consensus, leave...